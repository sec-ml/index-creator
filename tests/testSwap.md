| term                                                              | sub-term  | notes            | book | page             |
| ----------------------------------------------------------------- | --------- | ---------------- | ---- | ---------------- |
| this will be the term in this row, but sub term in the next row<> | yes, see? | example col swap | 1    | **10**, 1, 15-16 |
