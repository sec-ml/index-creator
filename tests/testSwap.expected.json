[
  {
    "term": "this will be the term in this row, but sub term in the next row",
    "subTerm": "yes, see?",
    "notes": "example col swap",
    "book": 1,
    "page": "1, **10**, 15-16"
  },
  {
    "term": "yes, see?",
    "subTerm": "this will be the term in this row, but sub term in the next row",
    "notes": "example col swap",
    "book": 1,
    "page": "1, **10**, 15-16"
  }
]
